pybind11/how-to.rst

```diff
@@ -401,57 +451,64 @@ The end result of all the steps above looks like this:
     #include "pybind11/pybind11.h"
     #include "pybind11/stl.h"
 
-    #include "numpy/arrayobject.h"
-    #include "numpy/arrayobject.h"
     #include "ndarray/pybind11.h"
 
     #include "lsst/pex/exceptions/python/Exception.h"
 
-    #include "lsst/TMPL/ExampleOne.h"
+    #include "lsst/tmpl/ExampleOne.h"
```

Comment:
"TMPL" is still used above on at least lines 177 and 184; this should be consistent.